src/Configuration/Defaults/Paths.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Codestyle\Configuration\Defaults;
+
+use Blumilk\Codestyle\Configuration\Paths as PathsContract;
+
+class Paths implements PathsContract
+{
+    protected array $paths = [];
+
+    public function __construct(string ...$paths)
+    {
+        $this->add(...$paths);
+    }
+
+    public function get(): array
+    {
+        return $this->paths;
+    }
+
+    public function add(string ...$paths): self
+    {
+        foreach ($paths as $path) {
+            $this->paths[] = $path;
```

Comment:
Maybe to avoid duplicates, lets check if value (path) exists in the array?
```suggestion
            if (!in_array($path, $this->paths) {
                $this->paths[] = $path;
            }
```

Reply:
https://github.com/blumilksoftware/codestyle/pull/9